### ISSUE TITLE
Replace Battery in MacBook Pro 13" Retina (Used by Mark) ?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#366]
Today I replaced the battery in my MacBook Air 11" and it's got a new life 🍄 
(_should be good for another couple of years!!_) 🎉 
@markwilliamfirth mentioned that the MacBook Pro he is using is warning about the battery... 😞 

![image](https://user-images.githubusercontent.com/194400/27555533-e8dfe884-5aa9-11e7-8ed3-30b22be9b969.png)

It is _possible_ but to change the battery but requires some effort and between **30 - 60 mins** of time ...
The cost of a "compatible" battery is **£46.99** which is _reasonable_ if it gives the laptop a "_second life_"... https://www.amazon.co.uk/Townpeak-Replacement-Battery-Warranty-Li-Polymer-x/dp/B013FKQVGS
![image](https://user-images.githubusercontent.com/194400/27555134-b615d586-5aa8-11e7-8229-ff2528c52a95.png)

> NOTE: from a bit of research it appears that changing the battery on the MacBook Pro Retina 13" (aka "MBPR13") is ***not*** as easy as it was for my MacBook Air 11" because the battery in the MBPR13 is ***glued*** to the **body**! 
Watch this video: https://www.youtube.com/watch?v=MQi5b084iXA
and read this overview: http://www.everymac.com/systems/apple/macbook_pro/macbook-pro-retina-display-faq/macbook-pro-retina-display-13-how-to-replace-battery.html

>@markwilliamfirth if you want the battery to be replaced (_which makes sense given the warning you're seeing ..._) please post Share the Serial Number and/or _exact_ Model Number of the Laptop so that we can confirm which battery to order from AMZN. 
> I'm _confident_ I can do it (_or we can ask @edwardcodes nicely to help; he has hardware XP_)
If you bring the new battery to PT we'll get it done. 👍

**Comment by nelsonic** [comment]
Paying someone `else` to do the replacement is ***£190*** ... 💸 
http://www.phoneheroeslondon.co.uk/iphone-repair-london/retina-a1425-a1502/battery-replacement/
![image](https://user-images.githubusercontent.com/194400/27555685-95f1fb84-5aaa-11e7-910e-2631a151b452.png)

You're welcome to look up alternative quotes, but I'm happy to do it if you can't find it _cheaper_. 👍

**Comment by markwilliamfirth** [comment]
My laptop's started dying when it reaches 12% now, so I will be looking into this

**Comment by nelsonic** [comment]
@markwilliamfirth we have the tools here (in PT) to do the switch so if you order the battery from Amazon and bring it with you we can do it. 👍

**Comment by markwilliamfirth** [comment]
<img width="270" alt="screen shot 2017-09-04 at 11 39 13" src="https://user-images.githubusercontent.com/5732209/30020877-bdd2f800-9165-11e7-84aa-3fc75b6a3db6.png">

Should do this when I'm back in London

**Comment by markwilliamfirth** [comment]
Laptop info is:

MBP13.3/2.5/8GB/256GBFLASH
PartNumber:MD213B/A
SerialNumber:C02JG38DDR54

Not sure I should do this myself for fear of breaking the laptop

Going to get a quote from Apple too

**Comment by markwilliamfirth** [comment]
Apple quoted me:

>Your MacBook comes with a 1 year warranty from the day of purchase. So it is currently out of warranty. However you can still have it repaired at a local Apple store for £129.

Going to try and file a consumer rights claim
https://www.apple.com/uk/legal/statutory-warranty/

**Comment by markwilliamfirth** [comment]
After explaining I wanted to claim under consumer law, they offered me a repair slot at:
MR SHOREDITCH, England, United Kingdom, EC2A 3DH

When I asked again if this would be free they said it would likely not be as to claim under consumer law I need to deal with Apple directly. They then booked me in for an appointment on Monday 2nd October at 5pm at Stratford City, but warned that Apple might not do the repair for free as a battery depleting could be considered fair use.

They also said I need to:
- Backup the laptop
- Bring ID
- Ensure I know my Apple ID + password

@iteles please note this means I might not have a laptop from 3rd October (until an unknown time) - do we have a backup machine I can use?

**Comment by iteles** [comment]
Would it not be less time consuming and hassle-free (just setting up a new laptop takes half a day) to wait a week and a half until we can get the tools to you to just replace the battery ourselves as suggested above? Or did you never end up ordering one from Amazon?

**Comment by markwilliamfirth** [comment]
The laptop isn't designed to be repaired - the battery is glued and I don't really fancy messing with it, plus apple might be free repair

**Comment by iteles** [comment]
Was this resolved @markwilliamfirth?

**Comment by markwilliamfirth** [comment]
@iteles no, I sensed you weren't happy with the decision so I cancelled the appointment

**Comment by markwilliamfirth** [comment]
The laptop dies without warning at 20% now

**Comment by iteles** [comment]
@markwilliamfirth This isn't an unsolveable impass.

**Comment by iteles** [comment]
@markwilliamfirth This isn't an unsolveable impass.

**Comment by iteles** [comment]
Closed from mobile erroneously 🙄 This UI needs some help!

This isn't an insolveable impass. Please book the appointment and spend the money (and time). I was merely suggesting it might be less hassle to do it as suggested. 

Please plan for tasks in that week that require the least setup, preferably browser-based (e.g. No changes to the website to review, get documentation of processes out of the way the weeek before etc) so you can use the chrome book  👌🏻

**Comment by markwilliamfirth** [comment]
@iteles where is the chromebook - how do I get hold of it?

**Comment by iteles** [comment]
I've been checking with a couple of people who were the last to use it and I believe it is no longer functional 😕 

We absolutely need a spare laptop for dwylers for eventualities such as this one so let's please order this one  (I did 10 mins of research here based on #177) on Prime so you have it when you need it:
https://www.amazon.co.uk/Acer-Chromebook-CB3-431-Celeron-Storage/dp/B01J3UVWKO

Linked to  https://github.com/dwyl/hq/issues/177 where decision was originally made to purchase one 👍

**Comment by markwilliamfirth** [comment]
Have ordered the laptop 👍

**Comment by iteles** [comment]
I haven't seen anything come into the office. 
Has this arrived?

Also, have you sorted out the laptop battery issue?

**Comment by Jbarget** [comment]
did this get sorted? i need to replace my battery and looking for recommendations of where to go.
Im not in warranty period anymore so doesnt necessarily need to be a apple licensed place, just real apple battery

**Comment by iteles** [comment]
Nope. The Chromebook was ordered 4 months after the issue was opened and arrived the next day, but 1 month later (by the time Mark left), nothing had been done.

@nelsonic Any recommendations for Justen on this?

**Comment by nelsonic** [comment]
@iteles I swapped out the battery in my MacBook Air for an OEM one and it's been working fine.
https://github.com/nelsonic/nelsonic.github.io/issues/371#issuecomment-311083652

There are a few more "_steps_" 
There are kits for the specific models: https://www.ifixit.com/Kits/Macbook-Pro-Retina-Battery-Kits
For example, _our_ MacBook Pro "13 Retina is this kit:
https://eustore.ifixit.com/en/Parts/MacBook-Parts/Macbook-Pro/13/Retina/Early-2013/MacBook-Pro-13-Retina-Late-2012-Early-2013-Battery-Fix-Kit.html
![image](https://user-images.githubusercontent.com/194400/34940758-b52ea208-f9e8-11e7-8e23-d83f5d49691c.png)

It requires about 1h of "work" according to the iFixit guide:
https://www.ifixit.com/Guide/MacBook+Pro+13-Inch+Retina+Display+Late+2012+Battery+Replacement/13076
I'm 100% "_up for it_" because I _enjoy_ doing DIY electronics.
But I can think there is merit in paying someone _else_ (£100) to do it _for_ us ...
https://github.com/dwyl/hq/issues/366#issuecomment-311152464 ⬆️  (_see above_)

@Jbarget what is your "_appetite_" for replacing it _yourself_...?
We could do a "battery replacement party" and get it done at the same time together ... 😉 
What is your "_urgency_"?

> Also, we could ask @yvonne-liu if there's any _discount_ we can get on the **iFixit** kit ... 🤔

**Comment by Jbarget** [comment]
@nelsonic sounds like a blast :)
id be looking to do it within the next couple of weeks if that works for you?

**Comment by yvonne-liu** [comment]
@Jbarget @nelsonic I can most definitely get you a discount 😄  hit me up. Repair is sexy.

**Comment by yvonne-liu** [comment]
Just checked, for EU123010-4, after my discount 😎  it's 55,63€ + 8,90€ shipping! I also have all the tools if you need to borrow them 💛 .

**Comment by nelsonic** [comment]
@Jbarget yeah, next couple of weeks is fine for me. An evening or weekend is best for me. You? 📆 
I need to check the serial number on the hardware to ensure I order the _correct_ battery. 🔍 
Will let you know. ⏳ 

@yvonne-liu thanks! I will _defs_ take you up on the kind offer of using your discount! ❤️

**Comment by Jbarget** [comment]
@nelsonic that sounds good, i'll work from FAC one day next week and i can hang around for the evening

@yvonne-liu thanks :)

**Comment by nelsonic** [comment]
Serial Number of the MacbookPro "13 is **C02KF8BBFFRP** and Model is **A1425**.
https://everymac.com/systems/apple/macbook_pro/specs/macbook-pro-core-i5-2.5-13-late-2012-retina-display-specs.html
![image](https://user-images.githubusercontent.com/194400/35234086-de259d60-ff97-11e7-9bdf-45382888746f.png)

@yvonne-liu can you please place an order and pass us the invoice so we can pay you back?
and does the kit from iFixit contain all the tools including the "adhesive remover" ...? 🤔 
(_happy to pay extra for the tools so we have "spare"_) 

Thanks! 🎉

**Comment by yvonne-liu** [comment]
@nelsonic @Jbarget ordered!

**Comment by iteles** [comment]
@yvonne-liu Please forward on the receipt so we can reimburse you!

**Comment by Jbarget** [comment]
@yvonne-liu thanks!
@iteles whatever is easiest re money let me know

**Comment by Jbarget** [comment]
scrap that
@yvonne-liu can you let me know how much i owe you too and to send me the same receipt :)

**Comment by yvonne-liu** [comment]
Update! They were out of stock so it was _just_ shipped via DHL, Tracking Number: 444789947084. I'll send an email to all re: ££

**Comment by yvonne-liu** [comment]
Update again! Package 1/2 was delivered today (just the batteries) and handed to @iteles, the fix kit is coming soon!

**Comment by yvonne-liu** [comment]
Hi! Sorry for the delayed response. I just checked, and unfortunately for @nelsonic the "fix kit" part is just completely out of stock and it's likely to be another week out 😟 ! But @Jbarget doesn't need a fix kit so maybe he can have a solo party 💃 ! Sorrryyyyyy.

**Comment by nelsonic** [comment]
Solo party indeed. 😉 
thanks for the update.

**Comment by yvonne-liu** [comment]
@nelsonic @Jbarget confirming that you both received all the parts needed!

**Comment by nelsonic** [comment]
@yvonne-liu thanks so much for your help! 🎉 
Please confirm you have received funds, else I can transfer using my new Monzo account (_thanks to JB!_)

**Comment by nelsonic** [comment]
Picking this up in PT. considering filming / timelapsing the work.
